### ISSUE TITLE
'themself' ?

### ISSUE THREAD

**Issue by mmao** [worrydream/EarlyHistoryOfSmalltalk#25]
"In this scheme (1 & 4) imply that classes are objects and that they must be instances of themself."

Is the quasi-singular possessive 'themself' intentional? In some way to be a specific concept for OOP classes?

**Comment by mmao** [comment]
Oh, I see in it the original scan. Is the goal to leave all typos present in the scan intact?

**Comment by worrydream** [comment]
Yes, we're trying to match the scan as much as possible.  Thank you!
